valhalla/sif/dynamiccost.h

```diff
@@ -696,6 +696,15 @@ class DynamicCost {
 
   virtual Cost BSSCost() const;
 
+  /*
+   * Determine whether an edge is currently closed due to traffic.
+   * @param  edgeid         GraphId of the opposing edge.
+   * @return  Returns true if the edge is closed due to live traffic constraints, false if not.
+   */
+  inline virtual bool IsClosed(const baldr::DirectedEdge* edge, const graph_tile_ptr& tile) const {
```

Comment:
hahah i just hate this. expanding the public part of the interface but there is no cleaner way unfortunately :frowning_face: